auto_round/mllm/autoround_mllm.py

```diff
@@ -34,6 +35,7 @@
 def _only_text_test(model, tokenizer, device):
     """Test if the model whether can use text-only datasets."""
     try:
+        device = detect_device(device)
```

Comment:
why need to add this, the device should have been  detected in autoround initialization

Reply:
this test func happened before autoround initialization